#include "game_server_player_replied_handler.h"
#include "thread_local/thread_local_storage.h"
#include "network/message_system.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void GamePlayerServiceRepliedHandler::Centre2GsLogin(entt::entity player,
	const ::Centre2GsLoginRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

void GamePlayerServiceRepliedHandler::OnCentreEnterGateSucceed(entt::entity player,
	const ::Centre2GsEnterGateSucceedRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}

